src/compounds/sponsored-product-rate-table/src/index.tsx

```diff
@@ -71,12 +73,19 @@ const ProductImage = ({ src, alt }: { src: string; alt: string }) => (
 
 interface UspTagsProps {
   usps: string[]
+  uspColor: string
+  beforeColor: string
 }
 
-const UspTags: React.FC<UspTagsProps> = ({ usps }) => (
+const UspTags: React.FC<UspTagsProps> = ({ usps, uspColor, beforeColor }) => (
   <React.Fragment>
     {usps.map((obj, index) => (
-      <UspTag usp={obj} key={`infoblock-${index}`} />
+      <UspTag
+        usp={obj}
+        backgroundColor={uspColor}
+        beforeColor={beforeColor}
+        key={`infoblock-${index}`}
```

Comment:
```suggestion
        key={index}
```

Key just needs to be _something_ unique to each USP, so you can use `index` directly